src/actions/TransactionExportActions.tsx

```diff
@@ -3,8 +3,13 @@ import { abs, add, div, gt, lt, mul } from 'biggystring'
 import csvStringify from 'csv-stringify/lib/browser/sync'
 import { EdgeCurrencyWallet, EdgeGetTransactionsOptions, EdgeTransaction } from 'edge-core-js'
 
+import { getExchangeDenomination } from '../selectors/DenominationSelectors'
+import { Dispatch, GetState } from '../types/reduxTypes'
+import { getHistoricalRate } from '../util/exchangeRates'
 import { DECIMAL_PRECISION } from '../util/utils'
 
+const UPDATE_TXS_MAX_PROMISES = 10
```

Comment:
Can this match the actual rates server limit of 100? If so let's import that constant

Reply:
Actually this is more of how many promises we want outstanding in the GUI. Has nothing to do with the 100 limit of the server.